Clinical-Genomics/cg
# adds upload delivery-report to scout cli

this PR adds CLI command to upload delivery-report to scout.

How to test:
0. install on stage (add instructions on how to as needed)

1. us
1. find case with report in hk and case exists in scout
1. Change the report that housekeeper points to 'housekeeper get - V CASE-ID'
1. cg upload delivery-report-to-scout CASE_ID

Expected outcome:
delivery-report CASE_ID uploaded to scout
